### PR TITLE
add persian language

### PR DESCRIPTION
**Pull request by alixsep**
add persian language

## Description
Added the persian language which is used for countries like Iran and Afghanistan and etc.
Also Fixed the bug that didn't show the files logo on custom language pages when the page was running on local storage.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -91,6 +91,11 @@ button {
   border-color: white;
   box-shadow: 0 1px 5px rgba(0, 0, 0, 0.2);
 }
+/***** Language Choices *****/
+/*
+.lang-options {}
+.lang-option {}
+*/
```
Comment (chingucoding) [inline css/custom.css:98]: Don't think we need them.

**Review by chingucoding — COMMENTED** [review]
Please remove the unused css-classes `lang-options` and `lang-option` from HTML and CSS :)

Also, I can't seem to find the fix for the icon not being shown on custom languages, did you forget to check it in?

File: css/custom.css (hunk 1 above)
Comment (alixsep) [reply css/custom.css]: I applied the changes!

**Review by chingucoding — APPROVED** [review]
Looks good to me, thank you for adding this!

**Pull request merged**